test/test-suite/src/main.rs

```diff
@@ -989,15 +991,15 @@ fn test_borrow_info() {
 
             // Borrow 0 is expected to be 16 bytes long and R/W.
             let info0 = caller.borrow(0).info().unwrap();
-            assert_eq!(
-                info0.attributes,
-                LeaseAttributes::READ | LeaseAttributes::WRITE
-            );
+            // assert_eq!(
```

Comment:
(this looks accidental, uncomment or delete the assert)

Reply:
As we discussed out of band, this was broken due to a missing PartialEq impl that I was gonna fix but hadn't yet. I've put it back.